custom_components/tuya_local/devices/yinmik_waterquality_tester.yaml

```diff
@@ -15,6 +15,20 @@ primary_entity:
       mapping:
         - scale: 10
 secondary_entities:
+  - entity: sensor
+    name: Total dissolved solids
+    icon: "mdi:water-opacity"
+    dps:
+      - id: 1
+        type: integer
+        name: sensor
+        class: measurement
+        unit: ppm
+        optional: true
+        mapping:
```

Comment:
Range should not be inside a mapping. min and max should be inside the range, however.

Reply:
Or since this is a read-only sensor, the range is not needed at all